app/src/main/java/com/xa/xpensauditor/TabFragment.java

```diff
@@ -58,7 +66,13 @@ public TabFragment() {
         // Required empty public constructor
     }
 
-
+    /**
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.SummaryJavadocCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>First sentence of Javadoc is missing an ending period.